libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/camera/DynamicCamera.java

```diff
@@ -70,7 +70,7 @@ public double zoom(RouteInformation routeInformation) {
    * Called when the zoom level should force update on the next usage
    * of {@link DynamicCamera#zoom(RouteInformation)}.
    */
-  public void forceResetZoomLevel() {
+  void forceResetZoomLevel() {
```

Comment:
Is this intentional? Do we want to make `forceResetZoomLevel` package-private? Even if this is only accessed internally, do 3rd party clients be interested in it?